### ISSUE TITLE
Copying updated diff alone to DB of different overpass instances(different servers)

### ISSUE THREAD

**Issue by arungowtham** [drolbr/Overpass-API#306]
I have multiple instances of overpass running in different machines, now i need to maintain uniform data in every machine, i cant individually update in every machine.

Is there a way to update in one machine and copy the diff to all other machines?
Can the diff alone be scraped out of updated overpass DB?

**Comment by mmd-osm** [comment]
I don't think this is possible at the moment. Read/write access is synchronized via `dispatcher` and that doesn't work across machines. The only option at this time is to run the update on every single machine.

**Comment by mmd-osm** [comment]
@arungowtham : did you maybe find some other solution in the meantime? Is this issue still relevant for you?
